### PR TITLE
v1.0 release

### PR DESCRIPTION
**Pull request by chwardle**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    "@tanstack/react-query-devtools": "5.17.21",
+    "@utahdts/utah-design-system": "1.14.1",
+    "@utahdts/utah-design-system-header": "1.14.1",
+    "clsx": "2.1.0",
+    "cz-conventional-changelog": "3.3.0",
+    "firebase": "9.23.0",
+    "firebase-admin": "11.11.0",
+    "glob": "10.3.10",
+    "graphql-request": "6.1.0",
+    "lint-staged": "15.2.0",
```
Comment (steveoh) [inline package.json:46]: **polish**: this is a dev dependency right?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    "@heroicons/react": "2.1.1",
+    "@hookform/error-message": "2.0.1",
+    "@hookform/resolvers": "3.3.4",
+    "@tanstack/react-form": "0.13.3",
+    "@tanstack/react-query": "5.17.19",
+    "@tanstack/react-query-devtools": "5.17.21",
+    "@utahdts/utah-design-system": "1.14.1",
+    "@utahdts/utah-design-system-header": "1.14.1",
+    "clsx": "2.1.0",
+    "cz-conventional-changelog": "3.3.0",
```
Comment (steveoh) [inline package.json:41]: **polish**: this is a dev dependency right?

File: public/robots.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# "We hold these truths to be self-evident,
+#          that all bots are created equal."
+#                      ~ Thomas Jeffersonbot
+User-agent: *
+Allow: *
+Crawl-delay: 1
+
+Sitemap: https://gps-billpay.utah.gov/sitemap.xml
```
Comment (steveoh) [inline public/robots.txt:8]: **thought**: this will need to update if it isn't the final dns

File: src/components/header/FooterMainContent.jsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            <ExternalLink href="https://gis.utah.gov/about/code/">GIS-related Utah Statute</ExternalLink>
+          </li>
+          <li>
+            <ExternalLink href="https://gis.utah.gov/about/policy/">UGRC Policies</ExternalLink>
+          </li>
+          <li>
+            <ExternalLink href="https://gis.utah.gov/about/media/">UGRC Media Resources</ExternalLink>
+          </li>
+          <li>
+            <ExternalLink href="https://gis.utah.gov/about/visiting-agrc/">UGRC Directions and Parking</ExternalLink>
```
Comment (steveoh) [inline src/components/header/FooterMainContent.jsx:72]: **todo**: these urls are all going to change in march

File: src/components/header/FooterSocialMedia.jsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        href="https://facebook.com/UtahAGRC"
+        className="icon-link"
+        target="_blank"
+        rel="noreferrer"
+      >
+        {IconsWebsite.IconFacebook()}
+        <span className="visually-hidden">UGRC Facebook, opens in a new tab</span>
+      </a>
+      <a
+        href="https://vimeo.com/utahagrc"
```
Comment (steveoh) [inline src/components/header/FooterSocialMedia.jsx:54]: **todo**: reminder to switch this to youtube as we migrate

File: src/components/pageElements/HomeLanding.jsx [hunk 6]
```diff
@@ -0,0 +1,132 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable max-len */
+// import pageUrls from '../routing/pageUrls';
+// import ProgressLog from './ProgressLog/ProgressLog';
```
Comment (steveoh) [inline src/components/pageElements/HomeLanding.jsx:4]: **chore**: remove unused imports

File: src/components/pageElements/Registration.jsx [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+        </div>
+        {busy && <Spinner id="spinner-registration-id" className="spinner spinner--indeterminate fullscreen" />}
+      </div>
+    )
+    : <Navigate to={pageUrls.home} />
+  );
+}
+
+// Registration.propTypes = propTypes;
+// Registration.defaultProps = defaultProps;
```
Comment (steveoh) [inline src/components/pageElements/Registration.jsx:279]: **chore**: remove unused code

File: src/components/pageElements/Subscription.jsx [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+      // eslint-disable-next-line no-console
+      console.error(`rolling back optimistic update with id ${context.id}`);
+    },
+    onSuccess: (successData, variables, context) => {
+      window.location.replace(successData.data);
+    },
+
+    onSettled: (settledData, error, variables, context) => {
+      // Error or success... doesn't matter!
+      // setBusy(false);
```
Comment (steveoh) [inline src/components/pageElements/Subscription.jsx:60]: **chore**: remove unused code

File: src/components/pageElements/Subscription.jsx [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+  return (
+    <div className="m-spacing-xl">
+      <h1 className="my-spacing-l text-center">TurnGPS Subscription</h1>
+
+      <div className="flex justify-end mb-spacing-l">
+        <Button
+          appearance="solid"
+          color="primary"
+          id="addSubscription"
+            // eslint-disable-next-line no-console
```
Comment (steveoh) [inline src/components/pageElements/Subscription.jsx:87]: **polish**: this seems unnecessary

File: src/components/pageElements/UtahIdProcess.jsx [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+
+          Creating the necessary accounts to access the TurnGPS bill pay system requires one simple step.
+        </p>
+
+        <div className="mt-spacing-l">
+          <h2 className="my-spacing-l">Get a UtahID Account</h2>
+
+          <p className="lead-in">
+            First, you will need to create a UtahID account{' '}
+            by going to <ExternalLink href="https://id.utah.gov/create?goto=https://turngps.utah.gov">id.utah.gov</ExternalLink>{' '}
```
Comment (steveoh) [inline src/components/pageElements/UtahIdProcess.jsx:31]: **question**: is the goto url correct?

File: src/css/2-tools/_tools-index.scss [hunk 11]
```diff
@@ -0,0 +1,4 @@
+/*
+############     _tools-index.scss      ############
+scss mixins and functions
```
Comment (steveoh) [inline src/css/2-tools/_tools-index.scss:3]: **suggestion**: remove empty/unused files

File: src/css/9-tip/_tip-index.scss [hunk 12]
```diff
@@ -0,0 +1,6 @@
+/*
+############     _tip-index.scss      ############
+A place to add css temporarily
+NOTE: CSS added this this file should eventually be removed or
+located to the correct folder/file where it best belongs.
+*/
```
Comment (steveoh) [inline src/css/9-tip/_tip-index.scss:6]: **suggestion**: remove unused/empty file

File: src/hooks/useUtahHeader.js [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+/**
+ * @param {string} url
+ * @param {import('react-router-dom').NavigateFunction} navigate
+ * @returns {EventAction}
+ */
+function actionFunctionForUrl(url, navigate) {
+  navigate(url);
+}
+
+export const authMenuItems = (hasUser, navigate) => (
```
Comment (steveoh) [inline src/hooks/useUtahHeader.js:23]: **question**: it seems this file needs some formatting fixes?

File: src/static/images/delicate-arch-tom-gainor-unsplash-06b92799.webp
Comment (steveoh) [inline src/static/images/delicate-arch-tom-gainor-unsplash-06b92799.webp:1]: **question**: is this file used?

File: src/static/images/kevin-quezada-Z8Ybwv9_v8M-unsplash.webp
Comment (steveoh) [inline src/static/images/kevin-quezada-Z8Ybwv9_v8M-unsplash.webp:1]: **question**: is this file used?

File: src/static/images/laptop-background-blur-f0b452f3.webp
Comment (steveoh) [inline src/static/images/laptop-background-blur-f0b452f3.webp:1]: **question**: is this file used?

File: src/static/images/turn_logo.png
Comment (steveoh) [inline src/static/images/turn_logo.png:1]: **question**: is it necessary to have the png and the svg? I would suggest using the svg everywhere.

**Review by steveoh — COMMENTED** [review]
looks good casey. a few questions and chores I think before we merge. Thoughts?

File: functions/index.js [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+import * as v1 from 'firebase-functions';
+import { https, setGlobalOptions } from 'firebase-functions/v2';
+import { params } from 'firebase-functions';
+import { expressServer } from './https/graphql/server.js';
+
+initializeApp();
+
+const vpc = params.defineString('VPC');
+const vpcEgress = 'ALL_TRAFFIC';
+const serviceAccount = 'firebase-function-v2-sa@ut-dts-agrc-turn-gps-dev.iam.gserviceaccount.com';
```
[the remark this answers is not in the document]
Comment (chwardle) [reply functions/index.js:12]: Is this the same email that is in the GitHub secrets SERVICE_ACCOUNT_EMAIL?

[the remark this answers is not in the document]
Comment (steveoh) [reply functions/index.js:12]: No that is the account deploying the app, but the `@projectid.iam` is in the project secret i believe and could be swapped out per environment.

File: src/components/pageElements/UtahIdProcess.jsx (hunk 10 above)
Comment (chwardle) [reply src/components/pageElements/UtahIdProcess.jsx]: I changed this to https://gps-billpay.utah.gov/

File: .github/workflows/push.yml [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+          VITE_FIREBASE_APPID: ${{ vars.VITE_FIREBASE_APPID }}
+          VPC: ${{ secrets.VPC }}
+
+  deploy-prod:
+    name: Deploy to production
+    runs-on: ubuntu-latest
+    needs: release-please
+    environment:
+      name: prod
+      url: https://gps-billpay.utah.gov
```
[the remark this answers is not in the document]
Comment (chwardle) [reply .github/workflows/push.yml:71]: I spoke with Sean and he said that URL works

File: src/components/header/FooterMainContent.jsx (hunk 4 above)
Comment (steveoh) [reply src/components/header/FooterMainContent.jsx:72]: @nathankota and @seanfernandez we'll have redirects for these but the mission is merging yada yada when the new website launches.

**Review by steveoh — APPROVED** [review]

**Pull request merged**